2016-03-13 12:56:23.510 [error] <0.1445.0> gen_server <0.1445.0> terminated with reason: no function clause matching inet_udp:send(#Port<0.3890>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>) line 60
2016-03-13 12:56:23.510 [error] <0.1445.0> CRASH REPORT Process <0.1445.0> with 0 neighbours exited with reason: no function clause matching inet_udp:send(#Port<0.3890>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>) line 60 in gen_server:terminate/7 line 826
2016-03-13 12:56:23.510 [error] <0.1440.0> Supervisor {<0.1440.0>,poolboy_sup} had child undefined started with {influx_udp_worker,start_link,undefined} at <0.1445.0> exit with reason no function clause matching inet_udp:send(#Port<0.3890>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>) line 60 in context child_terminated
2016-03-13 12:56:23.511 [error] <0.1327.0> gen_server <0.1327.0> terminated with reason: {{function_clause,[{inet_udp,send,[#Port<0.3890>,{127,0,0,1},80891,<<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1445.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:call/2 line 204
2016-03-13 12:56:23.511 [error] <0.1327.0> CRASH REPORT Process influx_udp with 0 neighbours exited with reason: {{function_clause,[{inet_udp,send,[#Port<0.3890>,{127,0,0,1},80891,<<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1445.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:terminate/7 line 826
2016-03-13 12:56:23.511 [error] <0.1434.0> gen_server influxdb_writer terminated with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3890>,{127,0,0,1},80891,<<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1445.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:call/2 line 204
2016-03-13 12:56:23.511 [error] <0.1434.0> CRASH REPORT Process influxdb_writer with 0 neighbours exited with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3890>,{127,0,0,1},80891,<<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1445.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:terminate/7 line 826
2016-03-13 12:56:23.512 [error] <0.1433.0> Supervisor rest_influxdb_sup had child undefined started with influxdb_writer:start_link() at <0.1434.0> exit with reason {{{function_clause,[{inet_udp,send,[#Port<0.3890>,{127,0,0,1},80891,<<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1445.0>,{write,[#{fields => ...,...},...]}]}},...} in context child_terminated
2016-03-13 12:56:23.512 [error] <0.1326.0> Supervisor influx_udp_sup had child undefined started with influx_udp:start_link() at <0.1327.0> exit with reason {{function_clause,[{inet_udp,send,[#Port<0.3890>,{127,0,0,1},80891,<<"test1 val=57 1457888183509000000\ntest val=23 1457888183509000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1445.0>,{write,[#{fields => ...,...},...]}]}} in context child_terminated
2016-03-13 12:56:23.673 [error] <0.1448.0> gen_server <0.1448.0> terminated with reason: no function clause matching inet_udp:send(#Port<0.3892>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>) line 60
2016-03-13 12:56:23.673 [error] <0.1448.0> CRASH REPORT Process <0.1448.0> with 0 neighbours exited with reason: no function clause matching inet_udp:send(#Port<0.3892>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>) line 60 in gen_server:terminate/7 line 826
2016-03-13 12:56:23.673 [error] <0.1440.0> Supervisor {<0.1440.0>,poolboy_sup} had child undefined started with {influx_udp_worker,start_link,undefined} at <0.1448.0> exit with reason no function clause matching inet_udp:send(#Port<0.3892>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>) line 60 in context child_terminated
2016-03-13 12:56:23.674 [error] <0.1449.0> gen_server <0.1449.0> terminated with reason: {{function_clause,[{inet_udp,send,[#Port<0.3892>,{127,0,0,1},80891,<<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1448.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:call/2 line 204
2016-03-13 12:56:23.674 [error] <0.1449.0> CRASH REPORT Process influx_udp with 0 neighbours exited with reason: {{function_clause,[{inet_udp,send,[#Port<0.3892>,{127,0,0,1},80891,<<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1448.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:terminate/7 line 826
2016-03-13 12:56:23.674 [error] <0.1447.0> gen_server influxdb_writer terminated with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3892>,{127,0,0,1},80891,<<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1448.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:call/2 line 204
2016-03-13 12:56:23.674 [error] <0.1447.0> CRASH REPORT Process influxdb_writer with 0 neighbours exited with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3892>,{127,0,0,1},80891,<<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1448.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:terminate/7 line 826
2016-03-13 12:56:23.675 [error] <0.1326.0> Supervisor influx_udp_sup had child undefined started with influx_udp:start_link() at <0.1449.0> exit with reason {{function_clause,[{inet_udp,send,[#Port<0.3892>,{127,0,0,1},80891,<<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1448.0>,{write,[#{fields => ...,...},...]}]}} in context child_terminated
2016-03-13 12:56:23.675 [error] <0.1433.0> Supervisor rest_influxdb_sup had child undefined started with influxdb_writer:start_link() at <0.1447.0> exit with reason {{{function_clause,[{inet_udp,send,[#Port<0.3892>,{127,0,0,1},80891,<<"test1 val=57 1457888183673000000\ntest val=23 1457888183673000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1448.0>,{write,[#{fields => ...,...},...]}]}},...} in context child_terminated
2016-03-13 12:56:23.846 [error] <0.1450.0> gen_server <0.1450.0> terminated with reason: no function clause matching inet_udp:send(#Port<0.3893>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>) line 60
2016-03-13 12:56:23.846 [error] <0.1450.0> CRASH REPORT Process <0.1450.0> with 0 neighbours exited with reason: no function clause matching inet_udp:send(#Port<0.3893>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>) line 60 in gen_server:terminate/7 line 826
2016-03-13 12:56:23.846 [error] <0.1440.0> Supervisor {<0.1440.0>,poolboy_sup} had child undefined started with {influx_udp_worker,start_link,undefined} at <0.1450.0> exit with reason no function clause matching inet_udp:send(#Port<0.3893>, {127,0,0,1}, 80891, <<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>) line 60 in context child_terminated
2016-03-13 12:56:23.846 [error] <0.1451.0> gen_server <0.1451.0> terminated with reason: {{function_clause,[{inet_udp,send,[#Port<0.3893>,{127,0,0,1},80891,<<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1450.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:call/2 line 204
2016-03-13 12:56:23.847 [error] <0.1451.0> CRASH REPORT Process influx_udp with 0 neighbours exited with reason: {{function_clause,[{inet_udp,send,[#Port<0.3893>,{127,0,0,1},80891,<<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1450.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:terminate/7 line 826
2016-03-13 12:56:23.847 [error] <0.1326.0> Supervisor influx_udp_sup had child undefined started with influx_udp:start_link() at <0.1451.0> exit with reason {{function_clause,[{inet_udp,send,[#Port<0.3893>,{127,0,0,1},80891,<<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1450.0>,{write,[#{fields => ...,...},...]}]}} in context child_terminated
2016-03-13 12:56:23.847 [error] <0.1452.0> gen_server influxdb_writer terminated with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3893>,{127,0,0,1},80891,<<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1450.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:call/2 line 204
2016-03-13 12:56:23.847 [error] <0.1452.0> CRASH REPORT Process influxdb_writer with 0 neighbours exited with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3893>,{127,0,0,1},80891,<<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1450.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:terminate/7 line 826
2016-03-13 12:56:23.847 [error] <0.1433.0> Supervisor rest_influxdb_sup had child undefined started with influxdb_writer:start_link() at <0.1452.0> exit with reason {{{function_clause,[{inet_udp,send,[#Port<0.3893>,{127,0,0,1},80891,<<"test1 val=57 1457888183845000000\ntest val=23 1457888183845000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1450.0>,{write,[#{fields => ...,...},...]}]}},...} in context child_terminated
2016-03-13 12:56:24.011 [error] <0.1453.0> gen_server <0.1453.0> terminated with reason: no function clause matching inet_udp:send(#Port<0.3894>, {127,0,0,1}, 80891, <<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>) line 60
2016-03-13 12:56:24.011 [error] <0.1453.0> CRASH REPORT Process <0.1453.0> with 0 neighbours exited with reason: no function clause matching inet_udp:send(#Port<0.3894>, {127,0,0,1}, 80891, <<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>) line 60 in gen_server:terminate/7 line 826
2016-03-13 12:56:24.011 [error] <0.1440.0> Supervisor {<0.1440.0>,poolboy_sup} had child undefined started with {influx_udp_worker,start_link,undefined} at <0.1453.0> exit with reason no function clause matching inet_udp:send(#Port<0.3894>, {127,0,0,1}, 80891, <<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>) line 60 in context child_terminated
2016-03-13 12:56:24.011 [error] <0.1454.0> gen_server <0.1454.0> terminated with reason: {{function_clause,[{inet_udp,send,[#Port<0.3894>,{127,0,0,1},80891,<<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1453.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:call/2 line 204
2016-03-13 12:56:24.012 [error] <0.1454.0> CRASH REPORT Process influx_udp with 0 neighbours exited with reason: {{function_clause,[{inet_udp,send,[#Port<0.3894>,{127,0,0,1},80891,<<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1453.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:terminate/7 line 826
2016-03-13 12:56:24.012 [error] <0.1326.0> Supervisor influx_udp_sup had child undefined started with influx_udp:start_link() at <0.1454.0> exit with reason {{function_clause,[{inet_udp,send,[#Port<0.3894>,{127,0,0,1},80891,<<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1453.0>,{write,[#{fields => ...,...},...]}]}} in context child_terminated
2016-03-13 12:56:24.012 [error] <0.1455.0> gen_server influxdb_writer terminated with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3894>,{127,0,0,1},80891,<<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1453.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:call/2 line 204
2016-03-13 12:56:24.012 [error] <0.1455.0> CRASH REPORT Process influxdb_writer with 0 neighbours exited with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3894>,{127,0,0,1},80891,<<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1453.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:terminate/7 line 826
2016-03-13 12:56:24.012 [error] <0.1433.0> Supervisor rest_influxdb_sup had child undefined started with influxdb_writer:start_link() at <0.1455.0> exit with reason {{{function_clause,[{inet_udp,send,[#Port<0.3894>,{127,0,0,1},80891,<<"test1 val=57 1457888184011000000\ntest val=23 1457888184011000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1453.0>,{write,[#{fields => ...,...},...]}]}},...} in context child_terminated
2016-03-13 12:56:24.181 [error] <0.1456.0> gen_server <0.1456.0> terminated with reason: no function clause matching inet_udp:send(#Port<0.3895>, {127,0,0,1}, 80891, <<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>) line 60
2016-03-13 12:56:24.181 [error] <0.1456.0> CRASH REPORT Process <0.1456.0> with 0 neighbours exited with reason: no function clause matching inet_udp:send(#Port<0.3895>, {127,0,0,1}, 80891, <<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>) line 60 in gen_server:terminate/7 line 826
2016-03-13 12:56:24.181 [error] <0.1457.0> gen_server <0.1457.0> terminated with reason: {{function_clause,[{inet_udp,send,[#Port<0.3895>,{127,0,0,1},80891,<<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1456.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:call/2 line 204
2016-03-13 12:56:24.181 [error] <0.1457.0> CRASH REPORT Process influx_udp with 0 neighbours exited with reason: {{function_clause,[{inet_udp,send,[#Port<0.3895>,{127,0,0,1},80891,<<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1456.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:terminate/7 line 826
2016-03-13 12:56:24.182 [error] <0.1458.0> gen_server influxdb_writer terminated with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3895>,{127,0,0,1},80891,<<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1456.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:call/2 line 204
2016-03-13 12:56:24.182 [error] <0.1458.0> CRASH REPORT Process influxdb_writer with 0 neighbours exited with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3895>,{127,0,0,1},80891,<<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1456.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:terminate/7 line 826
2016-03-13 12:56:24.182 [error] <0.1433.0> Supervisor rest_influxdb_sup had child undefined started with influxdb_writer:start_link() at <0.1458.0> exit with reason {{{function_clause,[{inet_udp,send,[#Port<0.3895>,{127,0,0,1},80891,<<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1456.0>,{write,[#{fields => ...,...},...]}]}},...} in context child_terminated
2016-03-13 12:56:24.182 [error] <0.1440.0> Supervisor {<0.1440.0>,poolboy_sup} had child undefined started with {influx_udp_worker,start_link,undefined} at <0.1456.0> exit with reason no function clause matching inet_udp:send(#Port<0.3895>, {127,0,0,1}, 80891, <<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>) line 60 in context child_terminated
2016-03-13 12:56:24.182 [error] <0.1326.0> Supervisor influx_udp_sup had child undefined started with influx_udp:start_link() at <0.1457.0> exit with reason {{function_clause,[{inet_udp,send,[#Port<0.3895>,{127,0,0,1},80891,<<"test1 val=57 1457888184180000000\ntest val=23 1457888184180000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1456.0>,{write,[#{fields => ...,...},...]}]}} in context child_terminated
2016-03-13 13:01:10.722 [error] <0.1460.0> gen_server <0.1460.0> terminated with reason: no function clause matching inet_udp:send(#Port<0.3896>, {127,0,0,1}, 80891, <<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>) line 60
2016-03-13 13:01:10.722 [error] <0.1460.0> CRASH REPORT Process <0.1460.0> with 0 neighbours exited with reason: no function clause matching inet_udp:send(#Port<0.3896>, {127,0,0,1}, 80891, <<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>) line 60 in gen_server:terminate/7 line 826
2016-03-13 13:01:10.722 [error] <0.1440.0> Supervisor {<0.1440.0>,poolboy_sup} had child undefined started with {influx_udp_worker,start_link,undefined} at <0.1460.0> exit with reason no function clause matching inet_udp:send(#Port<0.3896>, {127,0,0,1}, 80891, <<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>) line 60 in context child_terminated
2016-03-13 13:01:10.723 [error] <0.1461.0> gen_server <0.1461.0> terminated with reason: {{function_clause,[{inet_udp,send,[#Port<0.3896>,{127,0,0,1},80891,<<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1460.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:call/2 line 204
2016-03-13 13:01:10.723 [error] <0.1461.0> CRASH REPORT Process influx_udp with 0 neighbours exited with reason: {{function_clause,[{inet_udp,send,[#Port<0.3896>,{127,0,0,1},80891,<<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1460.0>,{write,[#{fields => ...,...},...]}]}} in gen_server:terminate/7 line 826
2016-03-13 13:01:10.723 [error] <0.1326.0> Supervisor influx_udp_sup had child undefined started with influx_udp:start_link() at <0.1461.0> exit with reason {{function_clause,[{inet_udp,send,[#Port<0.3896>,{127,0,0,1},80891,<<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1460.0>,{write,[#{fields => ...,...},...]}]}} in context child_terminated
2016-03-13 13:01:10.724 [error] <0.1459.0> gen_server influxdb_writer terminated with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3896>,{127,0,0,1},80891,<<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1460.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:call/2 line 204
2016-03-13 13:01:10.724 [error] <0.1459.0> CRASH REPORT Process influxdb_writer with 0 neighbours exited with reason: {{{function_clause,[{inet_udp,send,[#Port<0.3896>,{127,0,0,1},80891,<<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1460.0>,{write,[#{fields => ...,...},...]}]}},...} in gen_server:terminate/7 line 826
2016-03-13 13:01:10.724 [error] <0.1433.0> Supervisor rest_influxdb_sup had child undefined started with influxdb_writer:start_link() at <0.1459.0> exit with reason {{{function_clause,[{inet_udp,send,[#Port<0.3896>,{127,0,0,1},80891,<<"test1 val=57 1457888470722000000\ntest val=23 1457888470722000001\n">>],[{file,"inet_udp.erl"},{line,60}]},{influx_udp_worker,handle_call,3,[{file,"src/influx_udp_worker.erl"},{line,59}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]},{gen_server,call,[<0.1460.0>,{write,[#{fields => ...,...},...]}]}},...} in context child_terminated
